lambdas/sqs-message-remover/variables.tf

```diff
@@ -23,7 +23,7 @@ variable "lambda_processing_role_arn" {
 # Optional
 
 variable "cmr_custom_host" {
-  description = "Custom host to use for CMR requests"
+  description = "Custom protocol and host to use for CMR requests (e.g. http://cmr-host.com)"
   type        = string
   default     = ""
```

Comment:
default should be null, right?

Reply:
Is there a reason to not update `sqs-message-remover.tf`?

Reply:
Hmm. That should have came in when I cherry-picked the commit. Good catch. Let me take a look.